pkg/api/mlflow/dao/repositories/helpers.go

```diff
@@ -59,7 +59,7 @@ func BuildJsonCondition(
 		conditionTemplate = "%s->>? = ?"
 		idx := 0
 		for k, v := range jsonPathValueMap {
-			args[idx] = k
+			args[idx] = fmt.Sprintf("$.%s", k)
```

Comment:
It seems like this would require some other changes in the `query` module -- particularly in query.go, clause.go, query_test.go, etc

Reply:
thanks for letting me know! could you please elaborate on the required changes?

Reply:
for instance, the query parser is not supplying the '$.' prefix for keys, https://github.com/G-Research/fasttrackml/blob/main/pkg/api/aim/query/query_test.go#L283-L298

But the integration tests which depend on it are still passing. So I think it must be optional?

Reply:
```modified   pkg/api/aim/query/query.go
@@ -628,7 +628,7 @@ func (pq *parsedQuery) parseName(node *ast.Name) (any, error) {                                                                                                                                                                                                                                                                            Table: "contexts",                                                                                                                                                                                                                                                            Name:  "json",                                                                                                                                                                                                                                                        },                                                                                                                                                                                                   -                                                                        JsonPath:  contextKey,                                                                                                                                                                               +                                                                        JsonPath:  "$." + contextKey,                                                                                                                                                                                                                                                 Dialector: pq.qp.Dialector,                                                                                                                                                                                                                                           }, nil                                                                                                                                                                                                                                                                },
```

Reply:
![image](https://github.com/G-Research/fasttrackml/assets/1463452/57ea85ee-091e-42c2-8f87-0b8878468d08)

Maybe we should just make this change (and adjust query_test.go accordingly)?

Reply:
or assume that users will provide well-formed json path inputs?